tools/mount_gcsfuse/main.go

```diff
@@ -232,6 +232,16 @@ func run(args []string) (err error) {
 	// Run gcsfuse.
 	cmd := exec.Command(gcsfusePath, gcsfuseArgs...)
 	cmd.Env = append(cmd.Env, fmt.Sprintf("PATH=%s", path.Dir(fusermountPath)))
+
+        // Pass through the https_proxy/http_proxy environment variable,
```

Comment:
What happens if https_proxy is specified in both the env var and e.g. an entry in fstab?

Reply:
Do you mean something like this in fstab?

`my-bucket /my-mount-point gcsfuse rw,dir_mode=770,file_mode=660,https_proxy=http://my-proxy:8080`

Reply:
That's what I meant, but I guess the same thing happens if it's defined in the shell environment and you also specify it as an option with the CLI tool, seems to take environment variable value.

Reply:
Actually, at least on my linux, when appended to the fstab it gets as -o https_proxy=... to fusermount and fusermount fails with an unkown parameter exception.